hera_mc/cm_active.py

```diff
@@ -106,14 +107,17 @@ def load_connections(self, at_date=None):
         gps_time = self.set_times(at_date)
         self.connections = {'up': {}, 'down': {}}
         check_keys = {'up': [], 'down': []}
-        for cnn in self.session.query(partconn.Connections).filter((partconn.Connections.start_gpstime <= gps_time)
-                                                                   & ((partconn.Connections.stop_gpstime > gps_time)
-                                                                   | (partconn.Connections.stop_gpstime == None))):  # noqa
-            chk = cm_utils.make_part_key(cnn.upstream_part, cnn.up_part_rev, cnn.upstream_output_port)
+        for cnn in self.session.query(partconn.Connections).filter(
+            (partconn.Connections.start_gpstime <= gps_time)
+            & ((partconn.Connections.stop_gpstime > gps_time)
+               | (partconn.Connections.stop_gpstime == None))):  # noqa
```

Comment:
same comment regarding the `):` I'm not adamant about it, but would like some discussion at least.